setupDB.py

```diff
@@ -0,0 +1,93 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import Column, Integer, Float, Date, String, VARCHAR
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import csv
+import pandas as pd
+
+# app = Flask(__name__)
```

Comment:
Let's remove comments in the next commit. Since this file will have to be merged with main.py where the flaskapp is instantiated, it doesn't matter for now  :)